test/fixtures/seats.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```

Comment:
IMHO, for this small project fixtures won't be needed. We can also wait for others' opinions. Meanwhile, I would advice you to write a seed file.

Reply:
removed and seed added .thank you